packages/grant-explorer/package.json

```diff
@@ -131,7 +131,7 @@
     "tailwindcss": "^3.0.24"
   },
   "engines": {
-    "node": ">=16.15.0",
+    "node": ">=20",
```

Comment:
I see the current Node LTS version is 18, why is it necessary to upgrade?

Reply:
I see no reason not to upgrade, v18 will be LTS only by October '23 and v20 will be the new LTS by then. Unless something specifically requires us to stay on v18, I'd say let's keep it up to date. Not a strong opinion though, this is a minor thing.